### DISCUSSION TITLE
Use c8 for code coverage examples instead of nyc

### DISCUSSION THREAD

**Discussion by bompus** [avajs/ava#2842] [Ideas]
For AVA 4, I think it would be a good time to start recommending using c8 for coverage instead of nyc.

There are only a few instances in the codebase where it is mentioned: https://github.com/avajs/ava/search?q=nyc

**Comment by novemberborn** [comment]
Yea happy to take a PR that removes all mention of `nyc`. I use `c8` everywhere, it's not an AVA 4 thing.
